app/src/androidTest/java/ch/hikemate/app/ui/map/ZoomMapButtonTest.kt

```diff
@@ -34,9 +35,9 @@ class ZoomMapButtonTest {
       MapScreen(
           hikingRoutesViewModel = listOfHikeRoutesViewModel, navigationActions = navigationActions)
     }
-    composeTestRule.onNodeWithTag(ZOOM_MAP_BUTTON).assertIsDisplayed()
-    composeTestRule.onNodeWithTag(ZOOM_IN_BUTTON).assertIsDisplayed()
-    composeTestRule.onNodeWithTag(ZOOM_OUT_BUTTON).assertIsDisplayed()
+    composeTestRule.onNodeWithTag(ZoomMapButton.ZOOM_OUT_BUTTON).assertIsDisplayed()
+    composeTestRule.onNodeWithTag(ZoomMapButton.ZOOM_IN_BUTTON).assertIsDisplayed()
+    composeTestRule.onNodeWithTag(ZoomMapButton.ZOOM_OUT_BUTTON).assertIsDisplayed()
```

Comment:
Typo: I believe what you meant to write here is

```suggestion
    composeTestRule.onNodeWithTag(ZoomMapButton.ZOOM_MAP_BUTTON).assertIsDisplayed()
    composeTestRule.onNodeWithTag(ZoomMapButton.ZOOM_IN_BUTTON).assertIsDisplayed()
    composeTestRule.onNodeWithTag(ZoomMapButton.ZOOM_OUT_BUTTON).assertIsDisplayed()
```